README.md

```diff
@@ -32,4 +32,125 @@ The built in compiler remains solc 0.6, but users who want to use the newer comp
 ```
 npm install -D solc-0.7@npm:solc@^0.7.0
 npx solidity-docgen --solc-module solc-0.7
+```
+
+
+## Example Handlebars Formatting
+
+Each of the following examples will go through various methods exposed by the solidity-docgen library, how to use them in handlebars, and what the resulting markdown output would look like.
+
+1. Getting the natspec @dev and @notice tags
+
+### Handlebars formatting:
+
+``` {{{natspec.userdoc}}}
+    {{{natspec.devdoc}}}
+```
```

Comment:
GitHub doesn't let me leave a suggestion for this, but the first line of each of these code blocks is not being displayed in the Markdown rendering. The first line needs to be on its own line, like so:

<pre>
```handlebars
{{{natspec.userdoc}}}
{{{natspec.devdoc}}}
```
</pre>

If we add `handlebars` at the top it will probably display with some syntax highlighting.